dali/base/dafdesc.cpp

```diff
@@ -2336,7 +2336,29 @@ IFileDescriptor *createFileDescriptor(const char *lname,IGroup *grp,IPropertyTre
     return res;
 }
 
+IFileDescriptor *createFileDescriptor(const char *lname, const char *clusterType, const char *groupName, IGroup *group)
```

Comment:
very minor, what does lname stand for?

Reply:
The lname stands for logical name. This lname is copied from line 2316.